decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -65,6 +65,14 @@ def order
         @order = params[:order] || "random"
       end
 
+      def order_fields
+        @order_fields ||= begin
```

Comment:
And it's not exactly equivalent, as `defined?(@order_fields)` will return even if the value set is nil, whereas this type of memoization won't. Doesn't change anything in this scenario but I'd stick to ruby idioms if there's no appealing reason not to.